.storybook/stories/ConditionalReveal.tsx

```diff
@@ -0,0 +1,150 @@
+import { Meta } from '@storybook/react';
```

Comment:
Maybe these stories are not needed, since the pattern documentation covers this? What do you think @mikehazell